spec/lib/ability_spec.rb

```diff
@@ -110,6 +119,7 @@
 
     it_behaves_like "it can manage training requests"
     it_behaves_like "it can read notifications"
+    it_behaves_like "it cannot access problem reservations"
```

Comment:
This seems weird. A facility director should be able to access problem reservations, and it looks like they can through the interface, so I'm not sure what this test is doing.